tests/harness/python/harness.py

```diff
@@ -58,7 +58,8 @@ def unpack(message):
         result.Error = False
         result.AllowFailure = True
         result.Reason = repr(e)
+    sys.stdout = open(sys.stdout.fileno(), mode='w', encoding='utf8')
```

Comment:
Why is this needed?

Reply:
Otherwise sys.stdout uses an ascii codec writer, so I need to open it with a utf-8 output encoding. Removing it gives me:
```UnicodeEncodeError: 'ascii' codec can't encode character '\xff' in position 68: ordinal not in range(128)```

Reply:
Interesting. You could also maybe do explicit `encode` if it's only in a few places, not sure which is the least bad..